src/js/vre/chapter36/config/form.js

```diff
@@ -32,60 +32,46 @@ const formConfig = {
   title: 'Apply for vocational counseling',
   subTitle: 'Form 28-8832',
   defaultDefinitions: {
-    fullName
   },
   chapters: {
     applicantInformation: {
       title: 'Applicant Information',
+      initialData: {
+        socialSecurityNumber: '424242424'
+      },
       pages: {
         applicantInformation: {
           title: 'Applicant information',
           path: 'applicant-information',
           uiSchema: {
-            'view:isVeteran': {
-              'ui:title': 'Are you a Servicemember or Veteran applying for counseling service?',
+            socialSecurityNumber: ssnUI,
+            gender: {
+              'ui:title': 'Gender',
+              'ui:widget': 'radio'
+            },
+            seekingRestorativeTraining: {
+              'ui:title': 'Are you a child who is at least 14 years old, a spouse, or a surviving spouse with a disability and looking for special restorative training?',
+              'ui:widget': 'yesNo'
+            },
+            seekingVocationalTraining: {
+              'ui:title': 'Are you a child, a spouse, or a surviving spouse with a disability and looking for special vocational training',
               'ui:widget': 'yesNo'
             },
-            applicantFullName: _.merge(fullNameUI, {
-              first: {
-                'ui:required': formData => formData['view:isVeteran'] === false,
-              },
-              last: {
-                'ui:required': formData => formData['view:isVeteran'] === false,
-              },
-              'ui:options': {
-                expandUnder: 'view:isVeteran',
-                expandUnderCondition: false
-              }
-            }),
-            applicantRelationshipToVeteran: {
-              'ui:title': 'What is your relationship to the Servicemember or Veteran?',
-              'ui:widget': 'radio',
-              'ui:required': formData => formData['view:isVeteran'] === false,
-              'ui:options': {
-                expandUnder: 'view:isVeteran',
-                expandUnderCondition: false
-              }
```

Comment:
this stuff isn't in the mock up so I removed it.

Reply:
Ah, I think that's something I mixed up and isn't clear in the issue. The page you're working on is the second page int he applicantInformation chapter. So the existing config in here doesn't need to change. Sorry about that!